css/index.css

```diff
@@ -1 +1,376 @@
+}
+
+li {
+	list-style-type:disc;
+	margin-left: 18px;
+}
+.container {
+  max-width: 880px;
+  /* TB LR */
+  border: 1px solid red;
+  margin: 0 auto;
+}
+
+/* header */
+
+header {
+	display: flex;
+}
+
+
+header nav {
+  display: flex;
+  width: 100%;
+  vertical-align: top;
+  margin-top: 35px;
+}
+
+header nav a {
+  margin-right: 65px;
+  font-size: 1.4rem;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: flex;
+  margin-top: 25px;
+}
+
+.services-header-img {
+	margin-top: 30px;
+	margin-bottom: 30px;
+}
+
+.services {
+	padding-bottom: 10px;
+	border-bottom: 1px solid black;
+}
+
+/* cta */
+
+.cta {
+	display: flex;
+}
+
+.cta .cta-text {
+  vertical-align: top;
+  font-size: 7.7rem;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+	display: flex;
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  font-size: 1.6rem;
+  width: 410px;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  width: 100%;
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+	display: flex;
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  font-size: 1.6rem;
+  width: 275px;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  font-size: 1.6rem;
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 1.4rem;
+}
+
+.main-services-section {
+}
+
+.top-col {
+	display: flex;
+	vertical-align: middle;
+}
+
+.middle-col {
+	display: flex;
+}
+
+.last-col {
+	display: flex;
+}
+
+.our-services-col {
+	display: flex;
+}
+
+
+
+.item1 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item1 p {
+	padding-bottom: 20px;
+}
+
+.item2 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item2 p {
+	padding-bottom: 20px;
+}
+
+.item3 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
+}
+
+.item3 p {
+	padding-bottom: 20px;
+}
+
+.item4 {
+	border: 1px solid black;
+	height: 20px;
+	line-height: 20px;
+	margin: 20px;
+	padding: 20px;
```

Comment:
```suggestion
	padding: 20px 5% 20px 5%;
```

Reply:
when using a single parameter for padding it adds 40px on all sides and does use percentage base. I would do something like this